src/cli.rs

```diff
@@ -236,6 +238,24 @@ pub struct Img {
     ///approach the new image every frame.
     #[clap(long, env = "SWWW_TRANSITION_FPS", default_value = "30")]
     pub transition_fps: u8,
+
+    ///X cordinate to use as center for the transition
+    ///
+    ///Note only wokrs with the `from` transition
+    #[clap(long, env = "SWW_TRANSITION_CORDS_X", default_value = "0")]
```

Comment:
Wouldn't it be best to call this `COORDS` (i.e.: double `O`)? It's "coordinates".